src/main/java/edu/whimc/portals/Main.java

```diff
@@ -69,6 +67,11 @@ private void registerStuff() {
         pm.registerEvents(new PortalBlockChangeListener(), this);
         pm.registerEvents(new PortalDamageListener(), this);
 
+        // check if Citizens is enabled
+        if (pm.isPluginEnabled("Citizens")) {
```

Comment:
I have a few comments later about conditionally doing stuff in commands if Citizens is enabled so I'd store `pm.isPluginEnabled("Citizens")` as an instance variable in here for use across the plugin.